app/models/custom_button_set.rb

```diff
@@ -28,6 +28,14 @@ def update_children
     replace_children(children)
   end
 
+  def self.update_group_index(ids)
```

Comment:
Only thing I'm not sure about is the name...Since the caller API is named `reorder`, I think it would be clearer if this method was also called `reorder`